Dshop/apps/products_catalogue/api_views.py

```diff
@@ -0,0 +1,12 @@
+from .models import Product
+from .serializers import ProductSerializer
+from .filters import ProductFilter
+from rest_framework import viewsets
+from django_filters.rest_framework import DjangoFilterBackend
```

Comment:
PEP8 import order rules not followed

Reply:
It's not followed in the whole project. Good pointing out, we need to add something to guard id. Right now, let's ignore.